wpilib/wpilib/smartdashboard.py

```diff
@@ -380,12 +383,11 @@ def setDefaultBooleanArray(cls, key, defaultValue):
         return table.setDefaultBooleanArray(key, defaultValue)
 
     @classmethod
-    def getBooleanArray(cls, key, defaultValue):
+    def getBooleanArray(cls, key: str, defaultValue: T) -> Union[T, Tuple[bool]]:
```

Comment:
I just realised this return types are very, very wrong.

Reply:
What's wrong with them?

Reply:
The return value isn't necessarily a 1-tuple containing a bool.

Reply:
How is it not?

Reply:
@virtuald do you have any insight?

Reply:
If I set a NT to an array of 5 booleans, I wouldn't expect to get back a tuple containing only one boolean. Would you?

Reply:
Oh, I see what you mean. Sigh. 

[Looks like `Tuple[bool, ...]` would work?](https://github.com/python/mypy/issues/184)

Reply:
If you're fine with guaranteeing that a tuple is returned, that works.